packages/blip/test/unit/components/chart/deviceUsage.test.js

```diff
@@ -0,0 +1,215 @@
+/*
+ * == BSD2 LICENSE ==
+ * Copyright (c) 2017, Tidepool Project
```

Comment:
Copyright should be 2022? But this whole header seems weird to me. So are we supposed to use the Tidepool header in blip?